openwisp_controller/config/admin.py

```diff
@@ -559,28 +569,49 @@ def construct_change_message(self, request, form, formsets, add=False):
         return super().construct_change_message(request, form, formsets, add)
 
     def change_group(self, request, queryset):
+        # Validate all selected devices belong to the same organization
+        # which is managed by the user.
+        org_id = None
+        if queryset:
+            org_id = queryset[0].organization_id
```

Comment:
why not use the organization of the group here instead of doing this?

Reply:
When the form is rendered, there won't be any group selected. Hence, I used this.